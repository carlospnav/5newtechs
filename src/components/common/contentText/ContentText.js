import styled from "styled-components"
import css from "theme/variables"

export default styled.p`
  font-family: ${css.font.roboto};
  font-size: 20px;
  font-weight: ${css.fontWeight.regular};
  letter-spacing: 0.7px;
  color: ${css.color.sectionTitle};
`