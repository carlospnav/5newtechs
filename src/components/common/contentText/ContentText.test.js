import { shallow } from "enzyme"
import ContentText from "./ContentText"
import "jest-styled-components"

describe("<ContentText>", () => {
  const wrapper = shallow(<ContentText />)
  it("Should render a ContentText properly.", () => {
    expect(wrapper).toMatchSnapshot()
  })
})