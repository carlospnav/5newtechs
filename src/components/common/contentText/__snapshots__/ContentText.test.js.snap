// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentText> Should render a ContentText properly. 1`] = `
.c0 {
  font-family: 'Roboto',sans-serif;
  font-size: 20px;
  font-weight: 300;
  -webkit-letter-spacing: 0.7px;
  -moz-letter-spacing: 0.7px;
  -ms-letter-spacing: 0.7px;
  letter-spacing: 0.7px;
  color: #4e5760;
}

<p
  className="c0"
/>
`;
